config/languages/README.md

```diff
@@ -1,10 +1,15 @@
 # Languages
 
-The `languages.json` is a custom made file which contains languages from `languages_orig.json` that has `iso_639_2` standard defined.
+The `languages.json` is a custom made file which contains languages from `languages_orig.json` that has `iso_639_2` or `iso_639_3` standard defined.
 
-cd to this directory (`config/languages`) and type `node generate.js`
+To re-generate `languages.json`, run:
+
+```bash
+cd config/languages
+node generate.js
+```
 
 ### Todo
 
 - See [Missing languages](https://github.com/Trustroots/trustroots/issues/98)
-- If the search input for languages can one day handle 7000+ items, we can include all the living languages and dialects. Until that we're limiting this to ~500 or so.
+- If the search input for languages can one day handle 7000+ items performance wise, we can include all the living languages and dialects. Until that we're limiting this to ~500 or so.
```

Comment:
The new input in this PR should be capable enough, as long as we can async-load the list instead of including it out on every page load as we do now.